.github/workflows/cortex-arm.yml

```diff
@@ -4,15 +4,19 @@ on: [push, pull_request]
 
 jobs:
   coverity-scan:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
+      - uses: numworks/setup-arm-toolchain@2020-q4
       - name: checkout code
         uses: actions/checkout@v2
       - name: install dependencies
         run: |
-          sudo apt install gcc-arm-none-eabi zsh
+          sudo apt install zsh qemu-system-arm jq
```

Comment:
`jq` would be regarded as the enhancements, which can be decoupled from CI work.

Reply:
Undo it to minimize the necessary changes in single pull request.